reconcile/utils/terrascript_client.py

```diff
@@ -94,7 +94,7 @@
                  'variables', 'policies', 'user_policy',
                  'es_identifier', 'filter_pattern',
                  'specs', 'secret', 'public', 'domain',
-                 'aws_infrastructure_access']
+                 'aws_infrastructure_access', 'elasticsearch_version']
```

Comment:
We only add `elasticsearch_version` in the default file. These keys are used for values comes form gql queries, so we don't need to add `elasticsearch_version` here.